File: ForBusiness/VisualStudio/OLD/NCDataOnline/02/NCDataOnline/NCDataOnline/ConfigData.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace NCDataOnline
{
    public class ConfigData
    {
        // Fields
        public Dictionary<string, Dictionary<string, string>> conexao = new Dictionary<string, Dictionary<string, string>>();
        public Dictionary<string, Dictionary<string, string>> ficheiroJson = new Dictionary<string, Dictionary<string, string>>();
        public Dictionary<string, Dictionary<string, string>> ficheiroXls = new Dictionary<string, Dictionary<string, string>>();
        public Dictionary<string, string> geral = new Dictionary<string, string>();
        public string name = string.Empty;
        public Dictionary<string, Dictionary<string, string>> query = new Dictionary<string, Dictionary<string, string>>();

        // Methods
        public bool HasName()
        {
            return (this.name != string.Empty);
        }
    }


}

File: ForBusiness/NetBeans/NCDataOnlineService/src/ncdataonlineservice/ConfigData.java
/*
 * To change this license header, choose License Headers in Project Properties.
 * To change this template file, choose Tools | Templates
 * and open the template in the editor.
 */
package ncdataonlineservice;

import java.util.HashMap;
import java.util.Map;

/**
 *
 * @author tml
 */
public class ConfigData {
    public HashMap<String, HashMap<String, String>> conexao = new HashMap<String, HashMap<String, String>>();
    public HashMap ficheiroJson = new HashMap<>();
    public HashMap ficheiroXls = new HashMap<>();
    public HashMap geral = new HashMap<>();
    public HashMap<String, HashMap<String, String>> query = new HashMap<String, HashMap<String, String>>();
    public String name = "";
    
    public boolean HasName() {
        return (this.name != "");
    }
}
